packages/office-ui-fabric-react/src/utilities/color/_rgbaOrHexString.ts

```diff
@@ -0,0 +1,11 @@
+import { MAX_COLOR_ALPHA } from './consts';
+
+/**
+ * @internal
+ * Get a CSS color string from some color components.
+ * If `a` is specified and not 100, returns an `rgba()` string.
+ * Otherwise returns `hex` prefixed with #.
+ */
+export function _rgbaOrHexString(r: number, g: number, b: number, a: number | undefined, hex: string): string {
+  return a === MAX_COLOR_ALPHA || typeof a !== 'number' ? `#${hex}` : `rgba(${r}, ${g}, ${b}, ${a / MAX_COLOR_ALPHA})`;
```

Comment:
This is a pretty confusing function, why doesn't it always use the hex if always defined? Why is based off of the alpha? Especially since not all colors need the alpha.

Reply:
All this code is copied and pasted, all I did is move it. Improvements can/should be made in a subsequent PR.

Reply:
If a color has alpha, it can't be represented by 6-digit hex (and we don't support 8-digit hex with alpha on the end). The comments could possibly be improved, but the behavior is correct.

Reply:
(...technically 8 digit hex is supported by browsers, with the last two digits representing alpha)

Reply:
which is one of the reasons i'd like to remove colorValues.ts and substitute it in with the `getComputedStyles()` change, which is more flexible and resilient, and supports anything the browser supports, which to me makes more sense and would be the expected behavior of getting a color from a string